portfolio/src/main/webapp/script.js

```diff
@@ -40,13 +43,13 @@ function randomizeImage() {
 
 function showFrame() {
   document.getElementById('showiFrame').innerHTML =
-      '<iframe src="https://open.spotify.com/embed/track/1bSpwPhAxZwlR2enJJsv7U" width="300" height="380" frameborder="0" allowtransparency="true" allow="encrypted-media" ></iframe>';
+      '<iframe src="https://open.spotify.com/embed/track/1bSpwPhAxZwlR2enJJsv7U" width="100%" height="300px" frameborder="0" allowtransparency="true" allow="encrypted-media" ></iframe>';
 }
 
 function getComments() {
   fetch('/login').then((response) => response.text()).then((message) => {
```

Comment:
this arrow expression is long enough that you should probably move it into a named function